### DISCUSSION TITLE
Legendary never returns outputs or exits after uninstalling and reinstalling Python

### DISCUSSION THREAD

**Discussion by ghost** [legendary-gl/legendary#596] [Q&A]
I have uninstalled Python from my machine, and after that, all commands from legendary such as status and auth never return and just stay there empty.
After reinstalling python 3.11 from microsoft store it's the same. Legendary.exe should at least emit a warning if no python instance is found.

**Comment by CommandMC** [comment]
The `legendary.exe` executable available from the Releases page bundles its own Python.
Please make sure you are in fact using it (for example by running `where legendary`)
